contracts/cw-proposal-single/src/tests.rs

```diff
@@ -778,6 +778,82 @@ fn test_propose() {
     assert_eq!(created.id, 1u64);
 }
 
+#[test]
+fn test_propose_supports_stargate_message() {
+    let mut app = App::default();
+    let govmod_id = app.store_code(single_proposal_contract());
+
+    let threshold = Threshold::AbsolutePercentage {
+        percentage: PercentageThreshold::Majority {},
+    };
+    let max_voting_period = cw_utils::Duration::Height(6);
+    let instantiate = InstantiateMsg {
+        threshold: threshold.clone(),
+        max_voting_period,
+        only_members_execute: false,
+        allow_revoting: false,
+        deposit_info: None,
+    };
+
+    let governance_addr =
+        instantiate_with_cw20_balances_governance(&mut app, govmod_id, instantiate, None);
+    let governance_modules: Vec<Addr> = app
+        .wrap()
+        .query_wasm_smart(
+            governance_addr,
+            &cw_core::msg::QueryMsg::ProposalModules {
+                start_at: None,
+                limit: None,
+            },
+        )
+        .unwrap();
+
+    assert_eq!(governance_modules.len(), 1);
+    let govmod_single = governance_modules.into_iter().next().unwrap();
+
+    // Create a new proposal.
+    app.execute_contract(
+        Addr::unchecked(CREATOR_ADDR),
+        govmod_single.clone(),
+        &ExecuteMsg::Propose {
+            title: "A simple text proposal".to_string(),
+            description: "This is a simple text proposal".to_string(),
+            msgs: vec![CosmosMsg::Stargate {
```

Comment:
I added a [test](https://github.com/DA0-DA0/dao-contracts/pull/318/commits/63bb518d5cfc216e88355899d27729ec0513b267) that is breaking right now in the `cw-multi-test` package:
```
---- tests::test_execute_stargate_msg stdout ----
thread 'tests::test_execute_stargate_msg' panicked at 'Unsupported type', /.../cw-multi-test-0.13.4/src/untyped_msg.rs:54:18


failures:
    tests::test_execute_stargate_msg
```

Reply:
Ah, and a TODO cleanup when PR is merged - for a PR long-ago merged.

Can you make an issue on cw-plus to fix this up? I will try to do that today

Reply:
But it will need another release, which may take longer

Reply:
https://github.com/CosmWasm/cw-plus/pull/734

Reply:
I realized this will fix the parse error, but you still will get an error when the Stargate message is finally executed. We don't support this in multitest, neither Stargate nor ibc